### ISSUE TITLE
Different results between models with same K

### ISSUE THREAD

**Issue by jernesto87** [bstewart/stm#204]
Good afternoon,

more than an issue this is a question.
I have tried looking for the optimal K value using the Lee and Mimno algorithm (2014) and the initialization type set to "Spectral". However, when I try to use the K value obtained from the resulting model to generate a new model by explicitly setting up that value as initialization parameter, the outcome is a completely different model (despite the fact that I am still using Spectral as init.type). Therefore, I was wondering if the difference is directly related to the fact that the search K method (Lee and Mimno algorithm) also affects the training stage.

Thanks in advance for the clarification and regards.

**Comment by bstewart** [comment]
Yes- its actually a quite different spectral initialization.  I'm going to add a note to clarify this more- but the way the anchor words are chosen is completely different which means the solution is quite different.
